src/angular/app.js

```diff
@@ -212,8 +212,8 @@ controllers.controller('ResultsCtrl', ['$scope', 'Search', function ($scope, Sea
 
     // gets the activity details of the service 
     $scope.getActivityDetails = function(result){
-        //define a variable to store the 'closing time'
-        var activityDetails = null;
+        // Define a default variable value for activity details
+        var activityDetails = ["Unknown"];
```

Comment:
I don't know how the translation works right now but this won't get translated. I think just leave it blank if there aren't any